test/parallel/test-readline-interface.js

```diff
@@ -199,6 +199,27 @@ function isWarned(emitter) {
   assert.equal(callCount, expectedLines.length);
   rli.close();
 
+  // Emit two line events where there is a delay
+  //   between \r and \n
+  (() => {
```

Comment:
Can you drop this style and just wrap this portion of the test in curly braces. That will provide the block scope you want.